challenges/lib/src/test/java/challenges/InsertionSortTest.java

```diff
@@ -0,0 +1,28 @@
+package challenges;
+import challenges.utilities.InsertionSort;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class InsertionSortTest {
+
+    @Test

```

Comment:
Good work on your tests. Utilizing a negative number to see if that'd break the sort, nicely done!